package.json

```diff
@@ -2,46 +2,103 @@
   "name": "videojs-contrib-media-sources",
   "version": "2.4.3",
   "description": "A Media Source Extensions plugin for video.js",
-  "main": "videojs-media-sources.js",
+  "author": "Brightcove",
+  "license": "Apache-2.0",
+  "main": "src/plugin.js",
```

Comment:
This should be `"es5/plugin.js"` because we can't guarantee that people importing this package will be using ES6/Babel.